### PR TITLE
[sysdig, sysdig-deploy, node-analyzer] Feat/bench runner bump

### PR DESCRIPTION
**Pull request by cgeers**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped for the respective charts
- [x] Changelog updated for the charts with version updates.
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Comment by yashgiri** [comment]
Changes LGTM, will need to update chart versions because of a recent PR being merged of mine.
Probably need to improve the PR title as the change of automated changelogs is not merged/active yet.

File: charts/node-analyzer/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 This file documents all notable changes to Sysdig Node Analyzer Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## 1.8.4
+### Minor changes:
+* BenchmarkRunner
+    * bump to version 1.1.0.4
```
Comment (yashgiri) [inline charts/node-analyzer/CHANGELOG.md]: 1.1.0.5 right?

Comment (cgeers) [reply charts/node-analyzer/CHANGELOG.md]: nice catch. TY!

**Review by yashgiri — APPROVED** [review]
Thank you for the changes.

**Review by yashgiri — CHANGES_REQUESTED** [review]
It seems sysdig-deploy chart.yaml version change is missing will need to have that version change.

**Comment by cgeers** [comment]
> It seems sysdig-deploy chart.yaml version change is missing will need to have that version change.

Thanks. Missed this in the rebase -- fixed.

These versions being fast moving makes for an error prone process when several parties want to make a change. What can be done to improve this process?

**Review by lilx1ao — APPROVED** [review]

**Review by yashgiri — APPROVED** [review]

**Comment by yashgiri** [comment]
Hi, @cgeers because of this PR and some other PRs I do agree with your statement. Although I don't have a concrete answer, we have a sysdig deploy automatic chart update  workflow and automated changelog workflow(coming soon) that will improve the process. We can discuss more offline if need be.

**Pull request merged**